components/02-content/footnotes/footnotes.hbs

```diff
@@ -0,0 +1,6 @@
+<p>In juli 2018 is er een quick scan natuur uitgevoerd voor het plangebied<sup id="#voetnoot14link"><a href="#voetnoot14" class="dso-footnote-link">[14]</a></sup>. Dit onderzoek is een actualisatie van een eerder door Blom Ecologie uitgevoerd oriënterend onderzoek dat vanwege de datum van uitvoering haar geldigheid was verloren. In januari 2019 is dit onderzoek aangevuld<sup id="#voetnoot15link"><a href="#voetnoot15" class="dso-footnote-link">[15]</a></sup> met vrije kavel, welke eveneens onderdeel zijn van dit bestemmingsplan.</p>
```

Comment:
Kunnen we nog even nadenken over de class modifier `dso-footnote-link` ? Wikipedia doet het met reference:

![image](https://user-images.githubusercontent.com/9696261/75547306-dcef0680-5a2a-11ea-9f28-aea2b017b127.png)

Wat dacht je van `dso-footnote-reference` ? En zullen we de class op de `sup` zetten? Mochten we ooit nog de sup moeten stijlen, dan kunnen we dat